Packs/Active_Directory_Query/Integrations/Active_Directory_Query/connection_test.py

```diff
@@ -292,3 +292,18 @@ def add(self, *args, **kwargs):
     assert outputs.get('success') is False
     assert outputs.get('email') == 'test@paloaltonetworks.com'
     assert 'User must have a sAMAccountName' in outputs.get('errorMessage')
+
+
+def test_group_entry():
```

Comment:
add one more test from the `search_group_members` function using the group entry.
also, use specific custom attributes.